### ISSUE TITLE
Signing auth seems to be broken

### ISSUE THREAD

**Issue by PostScripton** [DIGITALCRIMINALS/OnlyFans#698]
Hi @DIGITALCRIMINALS, did I get everything right that auth with sign header is not working anymore? Because I get `{"error":{"code":401,"message":"Please refresh the page"}}` on trying to get a content from API.

What causes this error, what've been changed? Could you explain a bit?

**Comment by arjendevos** [comment]
Same here, seems the rules have updated.

`"format": "5714:{}:{:x}:636d3224",`
to:
`"format": "5750:{}:{:x}:63724ee3",`

Don't know where to look for the rest tbh

**Comment by PostScripton** [comment]
@arjendevos, I've looked [here](https://github.com/DIGITALCRIMINALS/dynamic-rules) but there was no changes for the last 5 days

**Comment by arjendevos** [comment]
@PostScripton yeah but OF has changed it. If i know where to look i could maybe update it but i have no idea where i can find those static params

**Comment by PostScripton** [comment]
@arjendevos But what about checksum indexes and all that stuff? Have they changed?

**Comment by arjendevos** [comment]
thats what i'm saying. It has to be updated.

**Comment by arjendevos** [comment]
@OFFriend can you explain how you retrieve the static params? Maybe i can build an automatic updater?

**Comment by DIGITALCRIMINALS** [comment]
Use this

https://github.com/DIGITALCRIMINALS/dynamic-rules/blob/main/onlyfans.json

**Comment by arjendevos** [comment]
@DIGITALCRIMINALS how many times does it get updated? And can I extract it myself?

**Comment by PostScripton** [comment]
@DIGITALCRIMINALS it seems to be changed again :D Therefore, the master branch is irrelevant

Current OF format string:
```
5768:{}:{:x}:6374bedd
```
Master branch of dynamic-rules:
```
"5764:{}:{:x}:6373986d"
```

**Comment by PostScripton** [comment]
@DIGITALCRIMINALS, dynamic-rules haven't been updated for the last week. What's the reason?

**Comment by rciptv2019** [comment]
@PostScripton Since it seems that this data is not being updated on time, could you tell how to remove it to put it manually?
Thank you

**Comment by PostScripton** [comment]
@rciptv2019 what do you mean?

**Comment by rciptv2019** [comment]
To the parameters of https://raw.githubusercontent.com/DIGITALCRIMINALS/dynamic-rules/main/onlyfans.json

I am trying to download content but when it does the data scraping and the download starts, it hangs without downloading anything and without giving an error.

**Comment by bytadaniel** [comment]
@DIGITALCRIMINALS the rules are completely broken. It has to be updated asap. Every route is returning 401 now
So, it would be very fine if you tell us how to generate onlyfans.json manually

**Comment by deviint** [comment]
New updated rules https://github.com/deviint/onlyfans-dynamic-rules

**Comment by iamdanylo** [comment]
Same thing {"error":{"code":401,"message":"Please refresh the page"}}. Maybe they changed it again
